cogs/games.py

```diff
@@ -133,6 +137,8 @@ def check(message: Message) -> bool:
         await ctx.send(embed=embed)
 
         while not guessed and tries > 0:
+            if ctx.author.id not in self.hangman_players:
```

Comment:
Hangman players should be limited to guild and channel because by playing hangman you can't chat normally (you'd get an invalid letter response). By making `self.hangman_players` a dict instead, and doing `self.hangman_players[ctx.guild.id][ctx.channel.id]` which will correspond to the list of players, you can avoid this.